### ISSUE TITLE
Why the norm layer (BN or IN) not be used in the network?

### ISSUE THREAD

**Issue by dawei03896** [hzwer/arXiv2020-RIFE#193]
Hi, Why the norm layer (BN or IN) not be used in the network? How would they affect the results?

**Comment by hzwer** [comment]
BN can make the network training more stable in the early stage, but it will increase some time overhead. Similar fields, such as optical flow estimation and super-resolution, also do not use BN.

IN I haven't tried much. Softsplat uses IN, which may have some effect.

**Comment by jzyustc** [comment]
@hzwer Hi, may I know how to add norm layers in RIFE if needed? 

I try to add BN by change the "Conv-ReLU" structure to "Conv-BN-ReLU" in each conv layers, but a significant PSNR drop is observed in validation of first 30 epochs(about 1-2 dB). 

I think it's caused by my wrong usage of BN, so could you please kindly provide your usage? Thanks!

**Comment by hzwer** [comment]
@jzyustc 
I have some observations：
 1. BN in the early stage of training makes the network effect worse, maybe just because the statistics change quickly, it does not mean that the model is not well fitted 
 2. When using BN, you can increase the learning rate of IFNet
3. If only IFNet is changed, there should be no problem in current implement

**Comment by jzyustc** [comment]
@hzwer Thanks for your careful replying!
